libs/hls.liq

```diff
@@ -6,10 +6,11 @@ let input.hls = ()
 # @param ~reload How often (in seconds) the playlist should be reloaded.
 # @param uri Playlist URI.
 # @flag experimental
-def input.hls.native(~id="",~reload=10.,uri)
+def input.hls.native(~id=null(),~reload=10.,uri)
   playlistr = ref([])
   sequence = ref(0)
   playlist_uri = ref(uri)
+  label_id = id ?? "input.hls.native"
```

Comment:
The correct way to do this is
```
id = string.id.default("input.hls.native", id)
```
If `id` is `null` it will generate an identifier of the form `input.hls.native_NNN` in the same way as usual sources do.

Reply:
Ok!